lib/jsdom/living/events/TouchEvent.webidl

```diff
@@ -1,9 +1,17 @@
+dictionary TouchEventInit : EventModifierInit {
+    sequence<Touch> touches = [];
+    sequence<Touch> targetTouches = [];
+    sequence<Touch> changedTouches = [];
+};
+
+[Constructor(DOMString type, optional TouchEventInit eventInitDict),
```

Comment:
Please note in the commit message that this also adds the TouchEvent constructor

Reply:
Well, I guess it won't work anyway, since we don't implement Touch instances or TouchList.